website/src/pages/index.tsx

```diff
@@ -12,67 +11,70 @@ import {
   feature3,
 } from "../components/landing/landing-images";
 import { Divider } from "../components/landing/divider";
+import { metadata } from "../metadata";
 
 export default function Home(): JSX.Element {
-  const { siteConfig } = useDocusaurusContext();
   return (
-    <Layout title={siteConfig.title} description={siteConfig.tagline}>
-      <div className="dark:bg-gray-500 bg-gray-200 dark:text-white text-theme-2">
-        <LandingHero
-          heading={siteConfig.title}
-          body={siteConfig.tagline}
-          copyText="npm install groqd"
-          navItems={[
-            { link: "/open-source/groqd/docs", title: "Documentation" },
-            { link: "/open-source/groqd/arcade", title: "Demo" },
+    <Layout>
```

Comment:
Main change in this file is just wrapping things in `<main>` so that Nuka Carousel doesn't show up in Google.  Hope that works 😄